### PR TITLE
Rails 6.1 Support

### PR DESCRIPTION
**Pull request by corasaurus-hex**
All tests pass and we have no deprecation warnings.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ jobs:
   rspec:
     parallelism: 1
     docker:
-      - image: cimg/ruby:2.7.2
+      - image: cimg/ruby:2.7.5
```
Comment (bhb) [inline .circleci/config.yml:12]: 👍

File: Gemfile.lock [hunk 2]
```diff
@@ -41,21 +41,20 @@ GEM
...
-    thread_safe (0.3.6)
-    tzinfo (1.2.10)
-      thread_safe (~> 0.1)
+    tzinfo (2.0.5)
+      concurrent-ruby (~> 1.0)
     zeitwerk (2.6.6)
 
 PLATFORMS
-  ruby
+  arm64-darwin-21
```
Comment (bhb) [inline Gemfile.lock]: Do you know why this changed? Our other `Gemfile.lock` files just keep the `ruby` platform

https://github.com/wunderteam/portal/blob/d552788a741017d4924288fa6d435c3a2590d1c6/Gemfile.lock#L843-L844

**Review by bhb — APPROVED** [review]
@corasaurus-hex Looks good. I have one question about whether it matters that we changed the platform in `Gemfile.lock`.

File: Gemfile.lock (hunk 2 above)
Comment (corasaurus-hex) [reply Gemfile.lock]: I'm not sure why it changed but I was able to use bundler to add the `ruby` platform and remove the `arm64-darwin-21` platform.

```
❯ bundle lock --add-platform ruby
Fetching gem metadata from https://rubygems.org/........
Resolving dependencies...
Writing lockfile to /Users/corasutton/Developer/wunder/nano-service/Gemfile.lock

❯ bundle lock --remove-platform arm64-darwin-21
Writing lockfile to /Users/corasutton/Developer/wunder/nano-service/Gemfile.lock
```

**Pull request merged**